src/screens/Auth/LoginStack/LoginScreen/LoginScreen.tsx

```diff
@@ -21,6 +22,35 @@ export default function LoginScreen(props: Props) {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
+  async function handleLogin(){
+    fetch(`${BASE_PATH}/api/auth/login`, {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        'Email': email,
+        'Password': password
+      }),
+    })
+      .then(res => res.json()
+      )
+      .then(data => {
+        console.log(data)
+        if(data.ID == undefined) {
+          Alert.alert('Incorrect email address or password')
```

Comment:
can you clear the password field if the email or password is incorrect. I see this commonly done on apps so I think it would be good for us to do.